### PR TITLE
COMMERCE-4698 prevent NPE when interacting with pricing layer

### PR DESCRIPTION
**Pull request by igorbeslic**
Please review

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
@igorbeslic tests looks failing with a valid reason

**Review by dsitu — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/price/CommerceProductPriceCalculationV2Impl.java [hunk 1]
```diff
@@ -744,80 +756,63 @@ private BigDecimal _getCommercePrice(
...
+		if (commerceTierPriceEntries.isEmpty()) {
+			return commercePrice;
+		}
 
-			BigDecimal currentPrice = commercePriceEntry.getPrice();
+		commercePrice = BigDecimal.ZERO;
 
-			currentPrice = currentPrice.multiply(
-				BigDecimal.valueOf(tierCounter));
+		CommerceTierPriceEntry commerceTierPriceEntry0 =
```
[the remark this answers is not in the document]
Comment (igorbeslic) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/price/CommerceProductPriceCalculationV2Impl.java:764]: Tnx updating the code

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
Merged already in diff pull

**Pull request closed without merging**